### ISSUE TITLE
Selected/Diff highlight is hard to read

### ISSUE THREAD

**Issue by ltpquang** [catppuccin/jetbrains#34]
Catppuccin plugin version: 2.0.3
IntelliJ IDEA 2022.2.3 - Build #IU-222.4345.14

This is screenshot from my project's Diff Viewer.
Do we need to strictly stick to Catppuccin's pallete?

![image](https://user-images.githubusercontent.com/4617357/201587450-fe1aba64-66c3-4796-9c1e-0efde5fc8482.png)

![image](https://user-images.githubusercontent.com/4617357/201587081-5ea4fa79-b07e-4c87-ba70-4252ccb2183e.png)

**Comment by kkrypt0nn** [comment]
No idea how you got that but I cannot reproduce it with any variant of the theme

### Frappé
<img width="1403" alt="image" src="https://user-images.githubusercontent.com/43011723/201588401-3cc0eee6-8710-4077-96da-31cbd8af467e.png">

### Macchiato
<img width="1430" alt="image" src="https://user-images.githubusercontent.com/43011723/201588457-99745a3b-81ea-4e97-95e5-5716c46c768f.png">

### Mocha
<img width="1403" alt="image" src="https://user-images.githubusercontent.com/43011723/201588502-537474ff-0419-4823-aadf-2f53cc285598.png">


> Do we need to strictly stick to Catppuccin's pallete?
That is the purpose of a palette and ports related to it, yep

**Comment by ltpquang** [comment]
@kkrypt0nn thank krypt, I get problem only with Frappe, other variants seem to be fine, could you please share your Color Scheme of DIff & Merge?

![image](https://user-images.githubusercontent.com/4617357/202081381-0ad0236b-624d-4026-8bec-d84be7140d9e.png)

**Comment by kkrypt0nn** [comment]
Sure thing, these are the settings for Frappé that I have after installing the plugin:
## Changed lines
### Changes
![image](https://user-images.githubusercontent.com/43011723/202208378-6f48359e-49b5-49f3-85c6-87cac9e128f7.png)

### Conflict
![image](https://user-images.githubusercontent.com/43011723/202208460-1af81fd7-b66c-495e-a147-323b267c3f2c.png)

### Deleted
![image](https://user-images.githubusercontent.com/43011723/202208510-5421dca4-512e-4de2-88b6-78d13083b76e.png)

### Inserted
![image](https://user-images.githubusercontent.com/43011723/202208586-9b7eaa3b-b604-4cd2-9ce4-e267564b67a9.png)

## Folded unchanged fragments
### Background
![image](https://user-images.githubusercontent.com/43011723/202208715-9ba5a9da-df8d-4e8a-90e6-96e38bf5face.png)


I've updated in the meantime to 2.0.4, maybe you can try to do that and see if it fixes your issue.

**Comment by ltpquang** [comment]
I noticed that the color scheme name of mine is different than yours, at first. It's was bold and violet.

![image](https://user-images.githubusercontent.com/4617357/202370815-0bf4431b-d2cf-4bc7-906b-b7bea63bc041.png)

I then tried to reset the scheme to default and everything is fine now.

![image](https://user-images.githubusercontent.com/4617357/202370775-99de57e7-e774-44e7-8d8e-427d5a39d786.png)

It's weird that uninstalling/reinstalling the plugin doesn't fix the scheme, we need to explicitly reset it to default.
Thanks for you support @kkrypt0nn. 🥳
